lexicons/com/atproto/admin/defs.json

```diff
@@ -93,16 +89,35 @@
     },
     "actionReversal": {
       "type": "object",
-      "required": ["reason", "createdBy", "createdAt"],
+      "required": ["createdBy", "createdAt"],
       "properties": {
-        "reason": { "type": "string" },
+        "comment": { "type": "string" },
         "createdBy": { "type": "string", "format": "did" },
         "createdAt": { "type": "string", "format": "datetime" }
       }
     },
+    "actionMeta": {
+      "type": "object",
+      "properties": {
+        "resolveReportIds": { "type": "array", "items": { "type": "integer" } },
+        "reportType": {
+          "type": "ref",
+          "ref": "com.atproto.moderation.defs#reasonType"
+        }
```

Comment:
I am interested to hear a little more about how to think about these two fields as a part of action metadata.

Reply:
`resolveReportIds` will go away since we don't need to keep track of individual reports being resolved.
`reportType` will stay and store the metadata for `#report` events only where the user specifies the `reportType` on the client.